sale_order_product_recommendation/models/sale_order.py

```diff
@@ -1,11 +0,0 @@
-# Copyright 2020 Tecnativa - Sergio Teruel
-# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
-
-from odoo import fields, models
-
-
-class SaleOrderLine(models.Model):
-    _inherit = "sale.order.line"
-
-    # Add database index to get last sale price query performance
-    product_id = fields.Many2one(index=True)
```

Comment:
Is this index not needed anymore?

Reply:
Hello @pedrobaeza, thank you for you in-depth review

Indeed, product_id is already indexed since v16: https://github.com/odoo/odoo/commit/318888fd835233dc17bb0d9f200544266a345ed4